packages/react-components/src/components/bar-chart/barChart.tsx

```diff
@@ -65,7 +65,6 @@ export const BarChart = (props: BarChartProps) => {
         [HOUR_IN_MS]: '1h',
         [DAY_IN_MS * 5]: '1d',
       },
-      // refreshRate,
```

Comment:
is refresh rate not supported on bar charts for now? probably worth noting in the doc site 😃

Reply:
It is. I believe this comment is an artifact of the previous way Sahil achieved refresh rate before moving it to the query.